x-pack/plugins/cloud_security_posture/common/constants.ts

```diff
@@ -5,10 +5,13 @@
  * 2.0.
  */
 
+import { QueryDslQueryContainer } from '@elastic/elasticsearch/lib/api/types';
 import { PostureTypes } from './types';
 
 export const STATUS_ROUTE_PATH = '/internal/cloud_security_posture/status';
 export const STATS_ROUTE_PATH = '/internal/cloud_security_posture/stats/{policy_template}';
+export const VULNERABILITIES_DASHBOARD_ROUTE_PATH =
```

Comment:
nit: it would be better to keep a certain convention between our dashboard's endpoints.
Right now `misconfiguration` dashboard is under `/stats` and `cnvm` dashboard is under `/dashboard` endpoint.

Personally, I prefer a more generalized naming convention, now it is being used by the dashboard but not necessarily it is what this endpoint is about going forward. 

I'm opening this to discussion

cc: @opauloh